share/spice/hackage/hackage.css

```diff
@@ -0,0 +1,3 @@
+.tile__body {
+    height: 10em;
```

Comment:
@cskksc looks like this is causing problems? It's cutting off the bottom of the tile for me:

![hackage_json_at_duckduckgo](https://cloud.githubusercontent.com/assets/873785/9174864/975b2846-3f50-11e5-9104-e7a33f1debfc.png)

Removing the CSS causes this to look OK:

![hackage_json_at_duckduckgo_and_1____projects_zeroclickinfo-goodies__ssh_](https://cloud.githubusercontent.com/assets/873785/9174884/b13ee9fa-3f50-11e5-811e-d7f2685151e6.png)